packages/performance/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "@ni-private/performance",
+    "version": "1.0.0",
+    "private": true,
+    "description": "A package to measure the performance of the Nimble Components",
+    "scripts": {
```

Comment:
For the build, performance, and start commands I made it so that each component that will be tested has their own commands. I don't think we want one enormous lighthouse test for every component. The chance that the components will interfere with each other would be higher. Instead each test can independently on their own page.